### ISSUE TITLE
How to create custom batches?

### ISSUE THREAD

**Issue by MichaelMMeskhi** [deepmind/jraph#48]
Assume I have multiple graphs (nodes, senders, receivers and graph label). How would one create batches that would work with `jax.vmap`?
